server/src/main/java/org/apache/druid/metadata/SQLMetadataConnector.java

```diff
@@ -327,6 +331,29 @@ tableName, getPayloadType()
         )
     );
   }
+  
+  public boolean tableContainsColumn(Handle handle, String table, String column)
+  {
+    try {
+      DatabaseMetaData databaseMetaData = handle.getConnection().getMetaData();
+      ResultSet columns = databaseMetaData.getColumns(
+          null,
+          null,
+          table,
+          column
+      );
+      return columns.next();
+    }
+    catch (SQLException e) {
+      return false;
+    }
+  }
+  
+  public void prepareTaskEntryTable(final String tableName)
+  {
+    createEntryTable(tableName);
```

Comment:
why not update createEntryTable to create the table who we want it going forward so no alert needs to be run every time a new cluster is coming online?

Reply:
@capistrant it was left as it as is since the subsequent update would have the same effect.
But yes, I'll modify the createEntryTable method. Thanks for the feedback

Reply:
Also wanted to confirm if you meant avoiding additional db calls to check and update the schema by "no alert needs to be run". Could you please clarify that?

Reply:
oops, I meant to type "alter" not "alert". I was just suggesting that newly built clusters only run CREATE TABLE and not create + ALTER TABLE. More of a nit comment than anything